CHANGELOG.md

```diff
@@ -23,6 +23,14 @@ Classify the change according to the following categories:
     ### Deprecated
     ### Removed
 
+## Develop - 2024-06-05
+### Fixed
+- Increased the big-M bound on maximum net metering benefit to prevent artificially low export benefits
+- Fixed a bug in which tier limits did not load correctly when the number of tiers vary by period in the inputs
+- Set a limit for demand and energy tier maxes to avoid errors returned by HiGHS due to numerical limits
+- Index utility rate demand and energy tier limits on month in addition to tier
```

Comment:
@zolanaj  I pulled the changes for this PR out under a new heading and added the last bullet. Could you please check that the added bullet is accurate, and maybe add a note about how this changes our assumptions compared to the previous behavior?

Reply:
Thanks for checking the documentation, especially as the PR evolved.  I made a small edit and compared to previous behavior here: https://github.com/NREL/REopt.jl/pull/401/commits/ccfcb512169f7927f48294416176dddd81229e3a